tests/foreman/ui/test_contentview.py

```diff
@@ -847,11 +846,45 @@ def test_negative_add_dupe_modules(self):
 
         @assert: User cannot add modules multiple times to the view
 
-        @caseautomation: notautomated
-
-
         @CaseLevel: Integration
         """
+        cv_name = gen_string('alpha')
+        module_name = 'samba'
```

Comment:
we have to add a module that is in the repo first, and then assert that we cannot add it again, think that in this case we cannot be independent from what the repo contain, if random module should be, it can be only from the ones that are in repo.